assets/src/components/parts/janus-capi-iframe/schema.ts

```diff
@@ -19,7 +19,7 @@ export const schema: JSONSchema7Object = {
 
 export const getCapabilities = () => ({
   configure: true,
-  save: true,
+  isCapi: true,
```

Comment:
I think as a capability it should just be `capi`

Reply:
Fixed